lib/archethic_web/live/chains/beacon_live.ex

```diff
@@ -135,4 +192,18 @@ defmodule ArchEthicWeb.BeaconChainLive do
     |> SummaryTimer.previous_summaries()
     |> Enum.sort({:desc, DateTime})
   end
+
+  defp register_to_beacon_pool_updates do
+    date = BeaconChain.next_summary_date(DateTime.utc_now())
+
+    Enum.map(BeaconChain.list_subsets(), fn subset ->
```

Comment:
I don't think you really need Enum.map , maybe Enum.each

Reply:
Ok. got it